plugins/chat/assets/javascripts/discourse/components/user-threads/index.gjs

```diff
@@ -71,39 +71,29 @@ export default class UserThreads extends Component {
   }
 
   <template>
-    <div class="chat__user-threads-container">
-      <div class="chat__user-threads" {{this.fill}}>
-        {{#each this.threadsCollection.items as |thread|}}
-          <div
-            class="chat__user-threads__thread-container"
-            data-id={{thread.id}}
-          >
-            <div class="chat__user-threads__thread">
-              <div class="chat__user-threads__title">
-                <ThreadTitle @thread={{thread}} />
-                <ChannelTitle @channel={{thread.channel}} />
-              </div>
+    <div class="dcp-user-threads" {{this.fill}}>
```

Comment:
should it be `dcp__` ?

Reply:
no, a prefix is part of the name of the block. otherwise it becomes confusing and it will seems dcp_ is the block, and "user-threads" the element

Reply:
ok